Gemfile

```diff
@@ -4,8 +4,9 @@ eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 # VMDB specific gems
 #
 
-gem "activerecord-deprecated_finders", "~>1.0.4",  :require => "active_record/deprecated_finders"
```

Comment:
I'm not sure I want deprecated_finders gone yet.
Is there a way we can bring these in for automate?

Or do we have a different plan?